pinot-common/src/main/java/com/linkedin/pinot/common/data/FieldSpec.java

```diff
@@ -43,7 +41,6 @@
  */
 @SuppressWarnings("unused")
 public abstract class FieldSpec implements Comparable<FieldSpec>, ConfigNodeLifecycleAware {
-  private static final Charset UTF_8 = Charset.forName("UTF-8");
 
```

Comment:
(OCD) Can you also remove the empty line followed by the removed line? Same for other files

Reply:
really? i was just deleting stuff that is not used....
oh well..